pkg/util/fs_util.go

```diff
@@ -175,7 +175,7 @@ func unTar(r io.Reader, dest string) ([]string, error) {
 		if err := extractFile(dest, hdr, tr); err != nil {
 			return nil, err
 		}
-		extractedFiles = append(extractedFiles, dest)
+		extractedFiles = append(extractedFiles, filepath.Join(dest, filepath.Clean(hdr.Name)))
```

Comment:
@loganprice Can you please add tests for it. 
- That would help prevent future breakage. 
- It would help reviewers understand the impact of the change. 

Thanks
Tejal

Reply:
Unit test have been added

Reply:
@tejal29 just curious to then PRs get reviewed? I added a unit test as requested.

Reply:
Looking now @loganprice